Header header
float32 distance
string id
int32 time
int16 groundTruth
stroll_bearnav/Feature[] feature
